eox_audit_model/audit_wrapper.py

```diff
@@ -0,0 +1,44 @@
+"""Wrapper file for eox-audit-model"""
+from functools import wraps
+
+from eox_audit_model.decorators import audit_method
+
+
+def audit_api_wrapper(action='', data_filter=None, hidden_fields=None):
```

Comment:
technically this is wrapper however IMO this makes much more sense in the file `eox_audit_model.decorators` with another name something like audit_django_api or audit_drf_api